src/Chapter 2/RemoveDups2.py

```diff
@@ -0,0 +1,89 @@
+"""2.1 Remove Dupes"""
+
+class Node(object):
+    """Node Class"""
+    def __init__(self, d, n=None):
+        self.data = d
+        self.next_node = n
+
+    @property
+    def next(self):
+        """Gets next node"""
+        return self.next_node
+
+    @next.setter
+    def next(self):
+        """Sets the next node"""
+        self.next_node = n
+
+    @property
+    def data(self):
+        """Gets value of node"""
+        return self.data
```

Comment:
props should refer to their private values not to the getter, may  cause an infinite loop since it calls itself 
`return self._data`